app/models.py

```diff
@@ -28,6 +28,7 @@ class User(Base):
     user_id = Column(String, primary_key=True, index=True)
     price_count = Column(Integer, nullable=False, server_default="0", index=True)
     created = Column(DateTime(timezone=True), server_default=func.now())
+    is_moderator = Column(Boolean, nullable=False, default=False)
```

Comment:
`server_default` is used elsewhere instead of `default`. maybe stick instead to what is currently done ?

Reply:
I tried, but it does not work when creating an user with crud.create_user(). I'm going to pass the  is_moderator field as an optional parameter for the crud.create_user() func.